### ISSUE TITLE
[HOLD for payment 2023-07-24] [$1000] Android - Attachment - Unable to scroll PDF files in preview

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#20867]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch New Expensify app
2. Go to any chat
3. Click on the PDF file
4. Scroll down the file

## Expected Result:
User is able to swipe to scroll down the file.

## Actual Result:
User is unable to scroll down the file, unless pinching in while scrolling is performed.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.28.3

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

https://expensify.testrail.io/index.php?/tests/view/3497642

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/bc876db3-b721-43c8-bca3-0e979a644b79

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~019bb267c27b2966de</li>
        <li>Upwork Job ID: 1669733353046675456</li>
        <li>Last Price Increase: 2023-06-23</li>
    </ul>
</details>

**Comment by mountiny** [comment]
given the size of the deploy and the urgency to get this out this is one of two/three remaining blockers. I am not sure if its big enough to hold on this if its just android and user can downlaod the PDF

**Comment by robertKozik** [comment]
I want to clarify the reasoning behind not passing the disabled state into react-native Pressable component. When pressable gets `disabled={true}` we are loosing the possibility to alter the cursor appearance on hovering (so we cannot show disabled cursor). This said I was opting for not passing disabled state directly, but to check the disabled state both, when passing, and also inside, the event handlers.

**Comment by abdulrahuman5196** [comment]
Will review this today

**Comment by greg-schroeder** [comment]
Bump @abdulrahuman5196!

**Comment by abdulrahuman5196** [comment]
@hoangzinh  proposal is working on android. But will check further to be double sure and provide review tomorrow.

**Comment by abdulrahuman5196** [comment]
@hoangzinh 's proposal here https://github.com/Expensify/App/issues/20867#issuecomment-1595795537 looks good.
And their hypothesis about Pressable and PanResponder is correct as well. Since we can't directly disable PressableWithFeedback as per this https://github.com/Expensify/App/issues/20867#issuecomment-1596867538. The current proposal is the best path ahead.

🎀👀🎀
C+ Reviewed

cc: @deetergp

**Comment by abdulrahuman5196** [comment]
Not overdue waiting on proposal confirmation by @deetergp  here https://github.com/Expensify/App/issues/20867#issuecomment-1604469050

**Comment by greg-schroeder** [comment]
PR review underway

**Comment by abdulrahuman5196** [comment]
Still in review

**Comment by abdulrahuman5196** [comment]
I don't think this issue should have penalty. We found an another issue during implementation and was discussing and waiting on @deetergp 's confirmation here - https://github.com/Expensify/App/pull/21714#issuecomment-1632650015

**Comment by abdulrahuman5196** [comment]
BZ Checklist:

>The PR that introduced the bug has been identified. Link to the PR:
The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:


https://github.com/Expensify/App/pull/20671#issuecomment-1646592304

>Determine if we should create a regression test for this bug.
If we decide to create a [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) for the bug, please propose the regression test steps to ensure the same bug will not reach production again.

No

**Comment by abdulrahuman5196** [comment]
@greg-schroeder BZ checklist is complete here https://github.com/Expensify/App/issues/20867#issuecomment-1646592428

**Comment by greg-schroeder** [comment]
Thanks @abdulrahuman5196! Taking care of payment now (not applying penalty/bonus)

**Comment by greg-schroeder** [comment]
Offer sent @hoangzinh

**Comment by abdulrahuman5196** [comment]
Accepted the offer @greg-schroeder

**Comment by greg-schroeder** [comment]
Payments complete:

@hoangzinh $1000 - C
@abdulrahuman5196 $1000 - C+

**Comment by greg-schroeder** [comment]
@hoangzinh @abdulrahuman5196 thoughts on the above?

**Comment by abdulrahuman5196** [comment]
@greg-schroeder This is not that the PDF not able to scroll., but the scrolling is not smooth. This issue of scroll being not smooth has been present for long time. I also noticed and mentioned the same during the review. The PDF is scrolling but clashing with horizontal and vertical scroll. We should create a separate issue to improve the scroll experience.

**Comment by greg-schroeder** [comment]
Okay, I thought as much. thanks
